internal/cmd/commands/authenticate/authenticate.go

```diff
@@ -35,5 +48,42 @@ func (c *Command) Help() string {
 }
 
 func (c *Command) Run(args []string) int {
-	return cli.RunResultHelp
+	client, err := c.Client(base.WithNoTokenScope(), base.WithNoTokenValue())
+	if c.WrapperCleanupFunc != nil {
+		defer func() {
+			if err := c.WrapperCleanupFunc(); err != nil {
+				c.PrintCliError(fmt.Errorf("Error cleaning kms wrapper: %w", err))
+			}
+		}()
+	}
+	if err != nil {
+		c.PrintCliError(fmt.Errorf("Error creating API client: %w", err))
+		return base.CommandCliError
+	}
+
+	// Lookup the primary auth method ID in the global scope
+	aClient := authmethods.NewClient(client)
+	pri, err := getPrimaryAuthMethodId(c.Context, aClient, "global", "")
```

Comment:
Nit: the scope package has a string for this
```suggestion
	pri, err := getPrimaryAuthMethodId(c.Context, aClient, scope.Global.String(), "")
```

Reply:
I didn't realize I had this option, thanks for cluing me in!